export function throwAnException() {
  throw new Error("An exception");
}
